### PR TITLE
Updates to Fakespot moment page [#14740]

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
We're now promoting Review Checker instead of the analyzer. This includes updates to copy, image, and CTA.

## Testing
http://localhost:8000/firefox/welcome/18/
https://www-demo9.allizom.org/firefox/welcome/18/

Should look good in both light mode and dark mode.

### CODE REVIEW

**Comment by craigcook** [comment]
Marking Do Not Merge while I wait for final signoff from stakeholders.

**Review by wen-2018 — APPROVED** [review]
The page looks good in light and dark mode. 👍 
Just one nitpick, I noticed the icon has some extra pixels. It is small enough on the page that it is not noticeable the way it is used. It doesn't need to block the launch.
icon

**Pull request merged**
